### ISSUE TITLE
How to convince your team to invest time on code improvement?

### ISSUE THREAD

**Issue by kentcdodds** [kentcdodds/ama#446]
> This question was asked via email

My question is probably very common for people working at startups. 

Most of the time we have to work on everything (frontend, backend, infrastructure, etc) and the pressure to release new features is super high. It might be the difference between closing more contracts and survive or having to declare bankrupt.

We have all sorts of experience levels in our team and our codebase started from a hackathon project. Now we have lots of moments when we want to change things but it is harder than it should be. The code is messy and there are no tests.

How can I convince the business people to invest engineer time on code quality and tests? They are 100% focused on new features and the project turning into a pile of fragile code.

Thanks a lot and may God bless you for your amazing work helping people

**Comment by jLouzado** [comment]
imo similar answer was given here: https://github.com/kentcdodds/ama/issues/440

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
